server/db/schema.ts

```diff
@@ -48,8 +48,8 @@ export const jobPostingsTable = pgTable('job_postings', {
   }),
   isPublished: boolean('is_published').default(false).notNull(),
   totalApplicants: integer('total_applicants').default(0).notNull(),
-  createdAt: timestamp('created_at').defaultNow().notNull(),
-  updatedAt: timestamp('updated_at').defaultNow().notNull(),
+  createdAt: timestamp('created_at', { withTimezone: true }).defaultNow().notNull(),
+  updatedAt: timestamp('updated_at', { withTimezone: true }).defaultNow().notNull(),
```

Comment:
`{withTimezone: true}` takes care that the timestamp stored in DB is in UTC format

Reply:
understood, did some research. give me some time to merge.
Question: what will happen to existing dates? which are already in database?
As per my research, data will still be the same, it's just `postgres` will assume they are in UTC. However, since server is in us-east, i.e. [UTC−05:00](https://www.google.com/search?sca_esv=ceb67ca9dc8f108c&sxsrf=ADLYWII8gqvnMrWEUqTJ74zCUTp_-cSMmQ:1726308872785&q=UTC%E2%88%9205:00&stick=H4sIAAAAAAAAAONgVuLUz9U3MIo3Nq9axModGuL8qGOSgamVgQEA1Fz3mRsAAAA&sa=X&ved=2ahUKEwjv7YnZmcKIAxU2zjgGHZYfITAQmxMoAHoECDQQAg) current timestamps are stored in that format. Now how do we deal with this? Will we have to write a [seed](https://github.com/profilecity/vidur/blob/main/server/utils/seeds.ts) which retrieves time, converts to UTC and saves it back?

Reply:
As per debugging, current timestamps are saved in UTC.